script/brave_app_bundle_utils.py

```diff
@@ -6,11 +6,9 @@
 # You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 
-# Here we can extend the list of non-base services that are allowed to be included in the app bundle.
+# Here we can extend the list of non-base services that are allowed to be
+# included in the app bundle.
 def extend_allowlisted_non_base_services(allowlisted_non_base_services):
     allowlisted_non_base_services.add(
         'com.brave.playlist.playback_service.VideoPlaybackService')
```

Comment:
@deeppandya we probably need to move it to the base module as well once feature is ready for release.

Reply:
Thank you @SergeyZhukovsky